include/xen/gnttab.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef __XEN_GNTTAB_H__
+#define __XEN_GNTTAB_H__
+
+#include <xen/public/grant_table.h>
+
+grant_ref_t gnttab_grant_access(domid_t domid, unsigned long mfn,
```

Comment:
It would be good to document the functions and what they are expected to return.

Reply:
Fixed, all functions in gnttab.h were documented.